### DISCUSSION TITLE
Initial configuration

### DISCUSSION THREAD

**Discussion by grzanka** [pytrip/pytripgui#475] [Q&A]
After installation of pytripgui user needs to configure it to be able to prepare treatment plans. This is complex operation. How should we simplify it ?

**Comment by grzanka** [comment]
Precalculate DDD and SPC files and store them somewhere under URL. Deployment on Windows its not easy, but Linux users may have a profit from automatically downloading an example set of SPC and DDD files.

**Comment by Yoanka** [comment]
For the benefit of Windows users, we could add a button on the Config screen or Config context menu to generate a remote config setup file from a Linux environment.
This file could be delivered to Windows users so that they can remotely connect to a Linux environment without pasting individual paths to files that they do not have direct access to from their system. This config file can even include user account data and an SSH key for the individual user, to maximally simplify the pytripgui setup process.
